docs/dev/connectors/filesystem_sink.md

```diff
@@ -117,11 +117,11 @@ input.addSink(sink);
 </div>
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
-val input: DataStream[Tuple2[IntWritable, Text]] = ...
+val input: DataStream[(IntWritable, Text)] = ???
 
-val sink = new BucketingSink[String]("/base/path")
-sink.setBucketer(new DateTimeBucketer[String]("yyyy-MM-dd--HHmm", ZoneId.of("America/Los_Angeles")))
-sink.setWriter(new SequenceFileWriter[IntWritable, Text]())
+val sink = new BucketingSink[(IntWritable, Text)]("/base/path")
+sink.setBucketer(new DateTimeBucketer("yyyy-MM-dd--HHmm"))
+sink.setWriter(new StringWriter[(IntWritable, Text)]())
```

Comment:
The problem is that `sink.setWriter` expects` Writer[(IntWritable, Text)]` and the `SequenceFileWriter` is of type `SequenceFileWriter[(IntWritable, Text), V]`

Reply:
@hequn8128 still won't work since the `setWriter` accepts Writer with a single type e.g T and the SequenceFileWriter accepts two types K,V. The tuple T is still a single type. The following will not compile :
```
val input: DataStream[(A, B)] = ???
  val sink = new BucketingSink[(A, B)]("/base/path")
  sink.setWriter(new SequenceFileWriter[A, B]())
```
because the SequenceWriter in this example should accept a tuple i.e `[(A,B)]` but of course it should also accept a V. i.e SequenceFileWriter[(A, B), V] which is not matched to the accepted type of `Writer[(A,B)]`

Reply:
@123avi Have you try java tuple2 for the example?  `org.apache.flink.api.java.tuple.Tuple2`. 

The code should be: 
`val input: DataStream[Tuple2[A, B]] = ???` 
not 
`val input: DataStream[(A, B)] = ???`

Reply:
@hequn8128 this is how you define a tuple in scala this (A,B) is syntactic of ` Tuple2[A, B]` is it exactly the same

Reply:
@123avi What I mean is don't use scala tuple. Use java tuple even for the scala example. 
`val input: DataStream[Tuple2[A, B]]` is different from `val input: DataStream[(A, B)]`. `org.apache.flink.api.java.tuple.Tuple2` is a class in Flink.

I wrote a sample code for you. Take a look at the code [here](https://github.com/hequn8128/flink/blob/bucketSinkTest/flink-connectors/flink-connector-filesystem/src/test/scala/org/apache/flink/streaming/connectors/fs/bucketing/ScalaBucketingSinkTest.scala#L62). You can also try to run the test. It works well.

Reply:
@hequn8128 sounds good, Thank you very much. I pushed the correction I did emphasize the usage of `java.tuple.Tuple2` to avoid the frustration of someone that will do copy&paste to the example :) .